code/game/objects/obj_defense.dm

```diff
@@ -90,7 +90,7 @@
 /obj/blob_act(obj/structure/blob/B)
 	if(isturf(loc))
 		var/turf/T = loc
-		if((T.intact && level == 1) || T.transparent_floor) //the blob doesn't destroy thing below the floor
+		if(level == 1 && (T.intact|T.transparent_floor)) //the blob doesn't destroy thing below the floor
```

Comment:
```suggestion
		if(level == 1 && (T.intact || T.transparent_floor)) //the blob doesn't destroy thing below the floor
```
Will both do the same due to how DM works. But I recon you wanted to do an OR here instead of a bitwise OR.